### PR TITLE
Feat: Get image status after composing with a goroutine on the background

### PR DESCRIPTION
**Pull request by biancarosa**

### CODE REVIEW

**Comment by biancarosa** [comment]
oh boy these test results are not looking good, i see data race and i'm sweating

File: pkg/images/images.go [hunk 1]
```diff
@@ -150,14 +150,19 @@ func Create(w http.ResponseWriter, r *http.Request) {
...
-	// 	for {
-	// 		image, _ := updateImageStatus(image, r)
-	// 		if image.Status != models.ImageStatusBuilding {
-	// 			break
-	// 		}
-	// 	}
-	// }()
+	go func(id uint) {
+		var i *models.Image
+		db.DB.First(&i, image.ID)
```
Comment (boaz0) [inline pkg/images/images.go:155]: shouldn't `image.ID` be replaced with `id`?

**Review by boaz0 — CHANGES_REQUESTED** [review]

File: pkg/images/images.go (hunk 1 above)
Comment (biancarosa) [reply pkg/images/images.go:155]: yes! you're a life savior

**Review by boaz0 — APPROVED** [review]

**Comment by boaz0** [comment]
LGTM feel free to merge it.

**Comment by maxamillion** [comment]
LGTM, merge!

**Review by maxamillion — APPROVED** [review]

**Pull request merged**